src/lib/toastr/toastr-config.ts

```diff
@@ -194,6 +198,7 @@ export const DefaultNoComponentGlobalConfig: GlobalConfig = {
   autoDismiss: false,
   newestOnTop: true,
   preventDuplicates: false,
+  countDuplicates: false,
```

Comment:
@yutamago @scttcper What do you think about enabling this by default? It emphasizes that a duplication was detected and may be a good default behavior. Also won't/shouldn't be triggered unless `preventDuplicates` and `resetTimeoutOnDuplicate` are enabled.